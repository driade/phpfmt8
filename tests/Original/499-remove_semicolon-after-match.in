<?php
//passes:AutoSemicolon
//version:8.0

$test  = 'a';
$array = [
    'value' => match ($test) {
        'a'     => 1,
        'b'     => 2,
    }
];