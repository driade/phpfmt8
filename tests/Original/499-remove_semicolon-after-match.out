<?php
//passes:AutoSemicolon
//version:8.0

$test = 'a';
$array = [
	'value' => match ($test) {
		'a' => 1,
		'b' => 2,
	},
];
$array = [
	'value' => match ($test) {
		'a' => 1,
		'b' => 2,
	},
];